tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_store_utils.ts

```diff
@@ -34,3 +34,20 @@ export function findFileIndex(
       item.file_path === fileSpec.file_path
   );
 }
+
+/**
+ * Find a range (with begin and end properties) in an array of ranges.
+ * @param range The ranges to search in.
+ * @param begin The begin of the range to search for.
+ * @param end The end of the range to search for.
+ * @returns Index (>=0) if found. -1 if not found.
+ */
+export function findRange(
```

Comment:
Few things about this util.

1. Word `Range` is overloaded as it can mean `Range` in HTML APIs. Prefer more concrete words
2. specify that the bounds are inclusive
3. return values should be boolean if the number greater or equal to zero is meaningless. If you change it, you should also change name of the util since `find` in JS API often mean return an instance (Range in this case)